### PR TITLE
Correct grammatical error "ran -> run"

### PR DESCRIPTION
**Pull request by aphirst**
#### What's changed?

All I've changed is a few grammatical typos. No code has been touched.

#### Where have you tested it?

I have not, but I have not touched the code!

*** 
#### Pull Request status
- [ ]  This PR has been tested using latest devkitPro, devkitARM, and EasyGL2D.  

_(Do not edit after this point)_
- [ ]  This PR is fully documented.
- [ ]  This PR has been tested before sending.
- [ ]  This PR follows C style and convention.

### CODE REVIEW

File: rungame/Makefile [hunk 1]
```diff
@@ -132,7 +132,7 @@ dist:	all
 $(TARGET).nds:	$(TARGET).arm7 $(TARGET).arm9
 	ndstool	-u 00030015 -g SLRN -c $(TARGET).nds -7 $(TARGET).arm7.elf -9 $(TARGET).arm9.elf \
 	python2 patch_ndsheader_dsiware.py $(TARGET).nds
-			-g SLRN 01 "TWLMENUPP-LR" -b icon.bmp "TWiLight Menu++;Last-ran ROM;Robz8"
```
Comment (legoj15) [inline rungame/Makefile]: It was grammatically correct before in this case

File: rungame/arm9/source/main.cpp [hunk 2]
```diff
@@ -107,7 +107,7 @@ std::string ReplaceAll(std::string str, const std::string& from, const std::stri
     return str;
 }
 
-int lastRanROM() {
```
Comment (legoj15) [inline rungame/arm9/source/main.cpp]: Ditto

File: rungame/arm9/source/main.cpp [hunk 3]
```diff
@@ -233,13 +233,13 @@ int main(int argc, char **argv) {
...
 
 	fifoWaitValue32(FIFO_USER_06);
-	if (fifoGetValue32(FIFO_USER_03) == 0) arm7SCFGLocked = true;	// If DSiMenu++ is being ran from DSiWarehax or flashcard, then arm7 SCFG is locked.
+	if (fifoGetValue32(FIFO_USER_03) == 0) arm7SCFGLocked = true;	// If DSiMenu++ is being run from DSiWarehax or flashcard, then arm7 SCFG is locked.
 
 	u16 arm7_SNDEXCNT = fifoGetValue32(FIFO_USER_07);
 	if (arm7_SNDEXCNT != 0) soundfreqsetting = true;
 	fifoSendValue32(FIFO_USER_07, 0);
 
-	int err = lastRanROM();
```
Comment (legoj15) [inline rungame/arm9/source/main.cpp]: same as above

File: titleandsettings/arm9/source/main.cpp [hunk 4]
```diff
@@ -186,7 +186,7 @@ void loadROMselect()
 	}
 }
 
-int lastRanROM()
```
Comment (legoj15) [inline titleandsettings/arm9/source/main.cpp]: ditto

File: titleandsettings/arm9/source/main.cpp [hunk 5]
```diff
@@ -496,7 +496,7 @@ int main(int argc, char **argv)
 
 	if (!ms().gotosettings && ms().autorun && !(keysHeld() & KEY_B))
 	{
-		lastRanROM();
```
Comment (legoj15) [inline titleandsettings/arm9/source/main.cpp]: same here

**Comment by aphirst** [comment]
Using "last run" as an adjective is equivalent to saying that that ROM is the last one which was run.

"was run" is the passive voice, which uses the past participle, not the simple past (which can be verified in reference to any textbook, or simply literature: but consider "When was it last done?" and "the last done one" vs "When was it last did?" and "the last did one"); and the past participle of "to run" is "run" ("ran" is the simple past).

**Comment by legoj15** [comment]
Ah well it's not up to me if this goes through, it's just that the functions look odd to me now.

**Pull request merged**